ovos_utils/log.py

```diff
@@ -361,8 +379,8 @@ def get_log_paths() -> Set[str]:
         set of paths to log directories
     """
     paths = set()
-    ALL_SERVICES.union({s.replace("-", "_") for s in ALL_SERVICES})
-    for service in ALL_SERVICES:
+    svc_names = ALL_SERVICES.union({s.replace("-", "_") for s in ALL_SERVICES})
```

Comment:
re comment about the default log list above, if i got the intention right, maybe this list could be a new optional kwarg in this method to cover usage by 3rd party libs, allowing for compat with any service names?

Reply:
I'm not sure `ALL_SERVICES` makes sense to track as a hard-coded list here anyway since everything else is generic and the values are only used to test for keys in configuration.. If this is intended to just get all of the log paths, I think iterating over configuration would be a better/more efficient method anywayss

Reply:
https://github.com/OpenVoiceOS/ovos-utils/pull/260/commits/89db8948c6ce13f0588efeea8e7b7f9161c62447 refactors to read from config and leaves no remaining references to `ALL_SERVICES`. This seems to address the intent of `get_log_paths` with some improved efficiency since it avoids calls to `get_log_path` that would just return the default path